tests/resources/transaction/test_pushtx.py

```diff
@@ -85,7 +85,7 @@ def test_get(self):
         self.assertFalse(data['success'])
 
         # force
-        response = yield self.web.get('push_tx', {b'hex_tx': bytes(tx.get_struct().hex(), 'utf-8'), b'force': b'true'})
+        response = yield self.web.post('push_tx', {'hex_tx': tx.get_struct().hex(), 'force': True})
```

Comment:
Shouldn't we keep the GET test until the support for GET is removed?

Reply:
@msbrogli as discussed in privately, I've done a refactor in the tests to have all tests for the GET and the POST. Can you see if it's good?